app/controllers/movements/TransportController.scala

```diff
@@ -44,19 +45,22 @@ class TransportController @Inject()(
 
   def displayPage(): Action[AnyContent] = (authenticate andThen getJourney(JourneyType.DEPART)) { implicit request =>
     val answers = request.answersAs[DepartureAnswers]
+    val consignmentReference = answers.consignmentReferences.map(_.referenceValue).getOrElse(throw ReturnToStartException)
```

Comment:
`.map(_.referenceValue).getOrElse(throw ReturnToStartException)` is quite repeating pattern over code. Maybe would be worth to extract it as method to answers/references class ?